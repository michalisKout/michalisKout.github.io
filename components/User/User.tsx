import PrimaryTitle from "../Shared/PrimaryTitle";
import UserContent from "./components/UserContent";
import SocialMedia from "./components/SocialMedia";
import UserImage from "./components/UserImage";

const User = () => {
  return (
    <>
      <section className="user--section">
        <div className="user--inner-container">
          <div className="user--inner-wrapper">
            <div className="lg:w-1/2">
              <PrimaryTitle />
              <UserContent />
              <SocialMedia />
            </div>
            <UserImage />
          </div>
        </div>
      </section>
    </>
  );
};

export default User;
