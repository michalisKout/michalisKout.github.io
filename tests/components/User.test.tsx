import { render } from '@testing-library/react';
import User from '../../components/User/User';

describe('User', () => {
  it('should render User', () => {
    const { asFragment } = render(<User imgUrl={'/'} mainContent={<div>content</div>} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
