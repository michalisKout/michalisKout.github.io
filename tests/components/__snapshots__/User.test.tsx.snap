// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should render User 1`] = `
<DocumentFragment>
  <section
    class="user--section"
  >
    <div
      class="user--inner-container"
    >
      <div
        class="user--inner-wrapper"
      >
        <div
          class="lg:w-1/2"
        >
          <h1
            class="user--title"
          >
            Who I am
          </h1>
          <section
            class="user--content"
          />
          <div
            class="user--social-medial--container"
          >
            <a
              aria-label="Twitter"
              class="mx-2"
              href="#"
            />
            <a
              aria-label="Facebook"
              class="mx-2"
              href="#"
            />
            <a
              aria-label="Linkden"
              class="mx-2"
              href="#"
            />
            <a
              aria-label="Github"
              class="mx-2"
              href="#"
            />
          </div>
        </div>
        <div
          class="user--image-container"
        >
          <div
            class="user--image-wrapper"
          >
            <div
              class="max-w-lg"
            >
              <img
                alt=""
                class="user--image"
                src="/"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
